src/java.base/share/classes/java/security/AccessController.java

```diff
@@ -853,14 +874,16 @@ private static PrivilegedActionException wrapException(Exception e) {
      */
     @CallerSensitive
     public static <T> T doPrivileged(PrivilegedExceptionAction<T> action,
-                                     AccessControlContext context, Permission... perms)
+                                     @SuppressWarnings("removal") AccessControlContext context, Permission... perms)
```

Comment:
you might find it easier if you put the Permissions parameter on a new line. There are several places in AccessController where the same thing happens.

Reply:
My rule is that a new line is added if there's only whitespaces before the insert position and the previous line does not end with a comma. In this case, unless I move `Permission... perms` onto a new line that an annotation on a new line looks like covering both `context` and `perms`.

Reply:
Turns out this only happens in this class:
```
$ rg '^\s*@SuppressWarnings\("removal"\).*?,.'
src/java.base/share/classes/java/security/AccessController.java:449:        @SuppressWarnings("removal") AccessControlContext context, Permission... perms) {
src/java.base/share/classes/java/security/AccessController.java:514:        @SuppressWarnings("removal") AccessControlContext context, Permission... perms) {
src/java.base/share/classes/java/security/AccessController.java:877:                                     @SuppressWarnings("removal") AccessControlContext context, Permission... perms)
```
I'll fix them manually.